pii-reporting-ui/src/app/features/app-shell/app-shell.component.css

```
@@ -0,0 +1,137 @@
+  background: var(--sentinel-bg-page) !important;
+  border-color: var(--sentinel-border-strong) !important;
+}
+
+/* Tab icons */
+.tab-icon {
+  margin-right: 0.5rem;
+}
+
+/* Scan Error Toast Styling
+   ::ng-deep needed: PrimeNG toast renders outside component scope */
+::ng-deep .p-toast.p-component {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: ng-deep

(lint/correctness/noUnknownPseudoElement)